dev-lang/python-oem/python-oem-3.9.16.ebuild

```diff
@@ -104,9 +68,7 @@ src_unpack() {
 }
 
 src_prepare() {
-	# Ensure that internal copies of expat and libffi are not used.
-	rm -r Modules/expat || die
-	rm -r Modules/_ctypes/libffi* || die
+	# Ensure that internal copies of zlib are not used.
```

Comment:
I think you can drop this comment from our modifications - looks like internal copy of zlib is no more.